tests/seq-properties.hs

```diff
@@ -596,7 +603,8 @@ prop_sortOn (Fun _ f) xs =
 
 prop_sortOnStable :: Seq UnstableOrd -> Bool
 prop_sortOnStable xs =
-    toList' (sortOn ordKey xs) ~= listSortOn ordKey (toList xs)
+    (fmap . fmap) unignore (toList' (sortOn ordKey xs)) ~=
+    fmap unignore (listSortOn ordKey (toList xs))
```

Comment:
No, this is wrong. You want

```haskell
Fun A UnstableOrd -> Seq A -> Bool
```

This means `UnstableOrd` will need an `Arbitrary` instance, which should be pretty easy to base on the one for pairs.